### DISCUSSION TITLE
Login Page Not Displaying in Wiki.js

### DISCUSSION THREAD

**Discussion by lucasalberti** [requarks/wiki#7347] [Error / Bug Report]
I was trying to adjust permissions so that guest users could view some pages from a specific path, while the rest of the pages would require a login in Wiki.js.

While testing and changing the settings, I was surprised to find that the login page suddenly disappeared, and now I can no longer access Wiki.js.

If anyone has a solution, I was making changes to the guest user group to allow them to see pages from a certain point onward. I suspect that might be the cause.

![wiki](https://github.com/user-attachments/assets/cd4a5a3f-b1a5-4b39-8e48-bf4b784de7fb)

**Comment by fnafox** [comment]
Try forcing it to show all login providers by appending `?all` to the end of the login url. `https://wiki.example.com/login?all`
